Library/Homebrew/extend/ENV/shared.rb

```diff
@@ -171,7 +171,11 @@ def compiler
 
   # @private
   def determine_cc
-    COMPILER_SYMBOL_MAP.invert.fetch(compiler, compiler)
+    if compiler == :brew_clang
+      "/usr/local/opt/llvm/bin/clang"
```

Comment:
Use `HOMEBREW_PREFIX`; can't assume `/usr/local`. Regardless: this definitely shouldn't be hardcoded here.